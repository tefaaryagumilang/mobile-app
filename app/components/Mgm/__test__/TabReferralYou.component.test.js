import React from 'react';
import renderer from 'react-test-renderer';

import TabReferralYou from '../TabReferralYou.component';

describe('TabReferralYou component', () => {
  it('TabReferralYou: renders correctly', () => {
    const tree = renderer.create(<TabReferralYou />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
