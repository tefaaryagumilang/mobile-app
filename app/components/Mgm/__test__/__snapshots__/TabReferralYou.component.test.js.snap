// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabReferralYou component TabReferralYou: renders correctly 1`] = `
<RCTScrollView>
  <View>
    <View>
      <View
        style={
          Object {
            "alignItems": "center",
            "paddingBottom": 20,
            "paddingTop": 20,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#FFFFFF",
              "borderColor": "#FFFFFF",
              "borderRadius": 12,
              "borderWidth": 2,
              "height": 25,
              "justifyContent": "center",
              "width": 25,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#ED1D25",
                "fontWeight": "700",
              }
            }
          >
            1
          </Text>
        </View>
        <View
          style={
            Object {
              "alignItems": "center",
              "paddingTop": 10,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#FFFFFF",
                "fontFamily": "Roboto",
              }
            }
          >
            Ini Kode
             
          </Text>
          <Text
            style={
              Object {
                "color": "#FFFFFF",
                "fontFamily": "Roboto",
                "fontWeight": "700",
              }
            }
          >
            Referal kamu
             
          </Text>
        </View>
      </View>
      <View>
        <Image
          source="MgmCode.png"
          style={
            Object {
              "alignSelf": "center",
              "height": 1224,
              "width": 598.4,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "paddingBottom": 20,
            "paddingTop": 30,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#FFFFFF",
              "borderColor": "#FFFFFF",
              "borderRadius": 12,
              "borderWidth": 2,
              "height": 25,
              "justifyContent": "center",
              "width": 25,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#ED1D25",
                "fontWeight": "700",
              }
            }
          >
            2
          </Text>
        </View>
        <View
          style={
            Object {
              "alignItems": "center",
              "paddingTop": 10,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#FFFFFF",
                "fontFamily": "Roboto",
              }
            }
          >
            Untuk membagikan kode kamu
             
          </Text>
          <View
            style={
              Object {
                "flexDirection": "row",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#FFFFFF",
                  "fontFamily": "Roboto",
                }
              }
            >
              tekan tombol
               
            </Text>
            <Text
              style={
                Object {
                  "color": "#FFFFFF",
                  "fontFamily": "Roboto",
                  "fontWeight": "700",
                }
              }
            >
              BAGIKAN
               
            </Text>
          </View>
          <Text
            style={
              Object {
                "color": "#FFFFFF",
                "fontFamily": "Roboto",
              }
            }
          >
             Kode Referal kamu siap untuk di bagikan.
             
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "paddingBottom": 30,
          }
        }
      >
        <Image
          source="MgmShare.png"
          style={
            Object {
              "alignSelf": "center",
              "height": 1224,
              "width": 598.4,
            }
          }
        />
      </View>
    </View>
  </View>
</RCTScrollView>
`;
